swagger_to_sdk_config.json

```diff
@@ -2,14 +2,13 @@
   "$schema": "https://openapistorageprod.blob.core.windows.net/sdkautomation/prod/schemas/swagger_to_sdk_config.schema.json",
   "meta": {
     "autorest_options": {
-      "version": "V2",
-      "use": "@microsoft.azure/autorest.python@~4.0.71",
+      "version": "3.0.6272",
+      "use": "@autorest/python@5.1.0-preview.1",
       "python": "",
       "python-mode": "update",
       "sdkrel:python-sdks-folder": "./sdk/.",
       "multiapi": "",
-      "keep-version-file" :"",
-      "no-async": ""
+      "track2"
```

Comment:
This isn't valid json and is causing pull requests azure-rest-api-specs to break. See #12080.